README.md

```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
 While adopting kafka at real.digital we noticed the immense entry barrier it poses to newcomers. 
 We can't recount how often we wrote Slack messages asking for the script to check the 
 status of topics or consumer groups. This is partly (but not only) due to a 
-fragmented and unclear definition of tooling and APIs for kafka. 
-In a wide array of administration tools, `esque` distances itself by striving to provide Kafka Ops for Humans, in a usable and natural way. We feel that the goal of `esque` embodies one of the Perl design principles: “**keep easy things easy, and make hard things possible**”. 
+fragmented and unclear definition of tooling and APIs for kafka.
+In a wide array of administration tools, `esque` distances itself by striving to provide Kafka Ops for Humans, in a usable and natural way.    
+  
+We feel that the goal of `esque` embodies the principle: “**keep easy things easy, and make hard things possible**”. 
 
 ## Principles
 
-Our driving objectives can be summed up through these principles:
-* comes with batteries included
+* batteries included
 * feature rich
 * robust
 * insightful
-* made by engineers for engineers
+* by engineers for engineers
 
-## Basic Features
+## Feature Overview
 
 * Support for any type of Kafka deployment >1.2
-* List Resources (Topics, Consumergroups, Brokers)
+* Listing of Resources (Topics, Consumergroups, Brokers)
```

Comment:
All the other sentences start with a verb. What about "List Resource Instances" or "Display Resources"

Reply:
```suggestion
* Display Resources (Topics, Consumergroups, Brokers)
```